src/plugins/home/public/application/components/manage_data/manage_data.tsx

```diff
@@ -23,6 +23,7 @@ import { createAppNavigationHandler } from '../app_navigation_handler';
 // @ts-expect-error untyped component
 import { Synopsis } from '../synopsis';
 import { getServices } from '../../kibana_services';
+/** @deprecated Use `RedirectAppLinks` from `@kbn/shared-ux-components */
```

Comment:
I don't think all of these comments are necessary: If you add `@deprecated` to the JSDoc comments where `RedirectAppLinks` is defined, as well as a `@removeBy` version, then the deprecation and all of its usages will automatically be tracked in the [deprecated API docs](https://docs.elastic.dev/kibana-dev-docs/api-meta/deprecations-due-by-team) (internal link).

Also, many IDEs will automatically indicate that the API in use is deprecated (vscode example below), so I don't think the comments are adding much.

<img width="706" alt="Screen Shot 2022-03-22 at 4 33 38 PM" src="https://user-images.githubusercontent.com/1608770/159587831-d39f21bf-8638-41dc-925c-c47bfbe27cbc.png">

Reply:
@rshen91 Just to clarify, I was recommending the following:
- Adding a deprecated (and optionally removeBy) flag [where RedirectAppLinks is defined](https://github.com/elastic/kibana/blob/843fc9bb0f846b8496ccaa09a80e3f4ef73518ea/src/plugins/kibana_react/public/app_links/redirect_app_link.tsx#L21-L38)
```diff
 * @remarks
 * It is recommended to use the component at the highest possible level of the component tree that would
 * require to handle the links. A good practice is to consider it as a context provider and to use it
 * at the root level of an application or of the page that require the feature.
+* 
+* @deprecated
+* @removeBy 8.3 // or whatever
 */
``` 
- Removing all of these comments (not just from this file, but from the whole PR)
```diff
-/** @deprecated Use `RedirectAppLinks` from `@kbn/shared-ux-components */
```  

This way you don't need any CODEOWNERS reviews, and we can use our existing docs tooling to determine where deprecated APIs are in use and track that they are removed on time. (Or, as Julia suggests, it might save y'all a lot of wrangling to just make those changes in this PR directly, since it's only used in a few places).